lms/djangoapps/instructor/views/api.py

```diff
@@ -2516,7 +2517,8 @@ def send_email(request, course_id):
     - 'subject' specifies email's subject
     - 'message' specifies email's content
     """
-    course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+    course_id = CourseKey.from_string(course_id)
+    course_overview = CourseOverview.get_from_id(course_id)
```

Comment:
You could move this closer to where it's being used, then you could avoid doing the work if `BulkEmailFlag.feature_enabled` is `False` -- eg, L2535